src/lib/checkbox/checkbox.md

```diff
@@ -25,8 +25,17 @@ remove the indeterminate state.
 
 ### Click action config
 When user clicks on the `mat-checkbox`, the default behavior is toggle `checked` value and set
```

Comment:
Notices just now that this sentence is missing a word- should be "the default behavior is to toggle..."